### ISSUE TITLE
Bad practice to set the global variables in Node.js

### ISSUE THREAD

**Issue by joetidee** [rstacruz/jsdom-global#29]
The JSDom documentation advises against making `window` and `document` , etc. part of the global namespace in Node.js. Should I therefore avoid using this package, or do you have a counter- argument?

**Comment by saivan** [comment]
Yes, I'm actually having issues with mocha randomly exiting during watch mode when this package is included. This seems dangerous to me.
